packages/types/src/request.ts

```diff
@@ -13,7 +13,7 @@ export interface Request {
   url?: string;
   method?: string;
   data?: any;
-  query_string?: string;
+  query_string?: string | { [key: string]: string };
```

Comment:
> The query string component of the URL. Can be given as unparsed string, dictionary, or list of tuples.

From [docs](https://develop.sentry.dev/sdk/event-payloads/request/#attributes). Wdyt about using a type/interface? Easier to read, we know where the dictionary is coming from...

Reply:
Good idea.